pom.xml

```diff
@@ -366,6 +367,11 @@
               </execution>
           </executions>
         </plugin>
+        <plugin>
```

Comment:
So each project needs to add the plugin to their "implementation" module right?

Reply:
Right. Adding it to the plugin section seems to be a non-starter just because the parent is used to build other things than implementation modules (like tests, other parent poms, etc.).

Reply:
Yeah, I realized that and it makes sense